index.js

```diff
@@ -18,21 +20,21 @@ module.exports = function (user, group) {
 		file.stat = file.stat || {};
 		file.stat.mode = file.stat.mode || defaultMode;
 
-		var finish = function () {
-			file.stat.uid = finalUid != null ? finalUid : file.stat.uid;
-			file.stat.gid = finalGid != null ? finalGid : file.stat.gid;
+		function finish() {
+			file.stat.uid = finalUid !== null ? finalUid : file.stat.uid;
+			file.stat.gid = finalGid !== null ? finalGid : file.stat.gid;
```

Comment:
No reason, but might be a good idea to not be strict since we aren't setting it to null anywhere.